<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TPProjet2Class</class>
 <widget class="QMainWindow" name="TPProjet2Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TPProjet2</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="status">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>70</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Status port : fermé</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ouvrirport">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Ouvrir le port</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>310</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Envoyer</string>
    </property>
   </widget>
   <widget class="QLabel" name="donnee">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Données reçues :</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_port">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Port :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_message">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>310</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_reception">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>261</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>534</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TPProjet2.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>ouvrirport</sender>
   <signal>clicked()</signal>
   <receiver>TPProjet2Class</receiver>
   <slot>onOpenPortButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onOpenPortButtonClicked()</slot>
  <slot>onSerialPortReadyRead()</slot>
  <slot>onSendMessageButtonClicked()</slot>
 </slots>
</ui>
